examples/discordjs-typescript/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "discordjs-typescript",
+  "version": "1.0.0",
+  "main": "dist/index",
+  "author": "PantheraRed",
+  "license": "MIT",
+  "scripts": {
+    "dev": "nodemon src/index.ts",
+    "build": "tsc",
+    "start": "node ."
+  },
+  "dependencies": {
+    "discord.js": "^12.5.3"
```

Comment:
Change djs to v13

Reply:
Not out yet but I'll make sure to commit those when v13 is officially out. :+1: